cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -270,3 +272,70 @@ This section comprises a glossary of all the keywords -- grouped by category and
+| <<administration-security-users-alter-password, ALTER CURRENT USER SET PASSWORD FROM ... TO>>                                                                                     | User and role administration  | Change the password of the user that is currently logged in.
+| <<administration-security-subgraph-show, SHOW [ROLE ... \| USER ... \| ALL] PRIVILEGES>>                                                                                          | User and role administration  | Returns information about role, user or all privileges.
+| <<administration-security-roles-grant, GRANT ROLE[S] ... TO>>                                                                                                                     | Privilege administration      | Assigns one or multiple roles to one or multiple users.
+| <<administration-security-administration-database-privileges, GRANT ... ON DATABASE ... TO>>                                                                                      | Privilege administration      | Assigns a database or schema privilege to one or multiple roles.
+| <<administration-security-subgraph-introduction, GRANT ... ON GRAPH ... [NODES \| RELATIONSHIPS \| ELEMENTS] ... TO>>                                                             | Privilege administration      | Assigns a graph privilege for one or multiple specified elements to one or multiple roles.
+| <<administration-security-administration-dbms-privileges-role-management, GRANT ... ON DBMS TO>>                                                                                  | Privilege administration      | Assigns a DBMS privilege to one or multiple roles.
+| <<administration-security-administration-database-privileges, DENY ... ON DATABASE ... TO>>                                                                                       | Privilege administration      | Denies a database or schema privilege to one or multiple roles.
+| <<administration-security-subgraph-introduction, DENY ... ON GRAPH ... [NODES \| RELATIONSHIPS \| ELEMENTS] ... TO>>                                                              | Privilege administration      | Denies a graph privilege for one or multiple specified elements to one or multiple roles.
+| <<administration-security-administration-dbms-privileges-role-management, DENY ... ON DBMS TO>>                                                                                   | Privilege administration      | Denies a DBMS privilege to one or multiple roles.
+| <<administration-security-roles-revoke, REVOKE ROLE[S] ... FROM>>                                                                                                                 | Privilege administration      | Removes one or multiple roles from one or multiple users.
```

Comment:
I think assigning and revoking roles go under role management instead of privilege management